packages/amesos2/test/adapters/Tpetra_CrsMatrix_Adapter_UnitTests.cpp

```diff
@@ -194,7 +194,7 @@ namespace {
     // create a Map
     const size_t numLocal = 10;
     RCP<Map<LO,GO,Node> > map = rcp( new Map<LO,GO,Node>(INVALID,numLocal,0,comm) );
-    RCP<MAT> eye = rcp( new MAT(map,1) );
+    RCP<MAT> eye = rcp( new MAT(map,numLocal,Tpetra::StaticProfile) );
```

Comment:
This matrix `eye` is the identity matrix; it has 1 entry per row, not 10.  The word "eye" comes from a Matlab function that creates an instance of the identity matrix.

Reply:
My matlab is rusty, so I didn't recognize that it was supposed to be the identity.